invisible_cities/database/load_db_test.py

```diff
@@ -178,3 +179,24 @@ def test_pmt_noise_frequencies(db):
     assert frequencies.shape[0] == db.nfreqs
     assert frequencies.shape[1] == db.feboxes + 1
     assert np.all(frequencies[:, 0] == freq_expected)
+
+
+def test_MixerDataNext100_max_version():
+    '''Check that MixerDataNext100 returns by default the data with maximum Version number'''
+
+    # check unique (G4Volume, Isotope)
+    df = DB.MixerDataNext100("Activity")
+    assert np.all(df.groupby(["G4Volume", "Isotope"]).count() == 1)
+
+    # check values correspond to maximum version number
+    conn = sqlite3.connect(DB.get_db("next100"))
+
+    query = '''SELECT * FROM Activity
+               WHERE G4Volume = '{0}' AND Isotope = '{1}'
+            '''
+
+    for (G4Volume, Isotope), act in df.groupby(["G4Volume", "Isotope"]):
+        act_ = pd.read_sql_query(query.format(G4Volume, Isotope), conn)
+        act_ = act_[(act_["Version"] == act_["Version"].max())]
+
+        np.testing.assert_allclose(act["TotalActivity"], act_["TotalActivity"])
```

Comment:
```suggestion
        np.testing.assert_allclose(act.TotalActivity, act_.TotalActivity)
```
Getting columns as attributes is more readable.